eng/pipelines/templates/analyze-api.yml

```diff
@@ -51,17 +54,14 @@ jobs:
       noCopyBinaries: true
       noDecompress: true
       isLargeApp: true
+      verbosityLevel: minimal
+      preserveTempFiles: true
```

Comment:
These were requested by the dev that is analyzing the APIScan run duration issues. They simply add more verbosity to the logs and keep additional files in our build artifacts.